### PR TITLE
Implemented the basic UI

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Created an app. rb file that will serve as your console app entry point. It should have methods that do the following:
  List all books.
  List all people.
  Create a person (teacher or student, not a plain Person).
  Create a book.
  Create a rental.
  List all rentals for a given person's id.
- Created main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method 
  should do the following:
  Present the user with a list of options to perform.
  Lets users choose an option.
  If needed, ask for parameters for the option.
  Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.interface
+end
+
```
Comment (OLIPLICHE) [inline main.rb]: - Hi, Can you work on your school library again? The requirements say your console app needs to behave the same as the [video example](https://www.youtube.com/watch?v=vkkgrhD6aXQ) provided for you.

Capture3

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Shaheryar0054, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Ongoing work in creating UI form (School library) 👍 

## Required Changes ♻️

- [ ] In ReadMe file: 
- The link to the PR you provided in your Readme, to clone the PR is not the proper one. Can you change it because it creates confusion?
- 
Capture
NB: The comment above is not in line with the code because you did not make the Readme visible 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (peterrobert) [reply main.rb]: This change has been implemented well

**Review by peterrobert — APPROVED** [review]
Hi @Shaheryar0054 , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work making the changes requested by the previous reviewer✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**